lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'menu'
+require_relative 'sendsms'
+
+class TakeAway
+
+  attr_reader :menu, :order
+
+  def initialize (menu = Menu.new)
+    @menu = menu
+    @order = Hash.new(0)
+    # @basket = Array.new
+  end
+
+  def menu_list
+    @menu.show_menu
+  end
+
+  def add_meal item, quantity = 1
+    fail 'dish not in the menu' if !menu.dishes.include?(item)
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)